src/gwt/src/org/rstudio/studio/client/workbench/views/console/ConsolePane.java

```diff
@@ -121,6 +122,20 @@ public IsWidget getProfilerInterruptButton()
       return profilerInterruptButton_;
    }
 
+   @Override
+   public IsWidget getSuspendBlockedIcon() { return consoleSuspendBlockedIcon_; }
+
+   @Override
+   public IsWidget setSuspendBlockedIcon(SessionSuspendBlockedEvent event) {
+      consoleSuspendBlockedIcon_.setVisible(event.isBlocked());
+      consoleSuspendBlockedIcon_.setTitle(event.getMsg());
+
+      return consoleSuspendBlockedIcon_;
+   }
+
+   @Override
+   public IsWidget getSuspendedIcon() { return consoleSuspendedIcon_; }
+
```

Comment:
NOTE: these require imports, and I have not added those to the suggestions specifically.

Example of how this would change based on the other changes:
```suggestion
   @Override
   public void onSerializationEvent(SessionSerializationEvent event) {
      int eventType = event.getAction().getType();
      switch (eventType) {
         case SessionSerializationAction.SUSPEND_SESSION:
            consoleSuspendBlockedIcon_.setVisible(false);
            consoleSuspendedIcon_.setVisible(true);
            break;
         case SessionSerializationAction.RESUME_SESSION:
            consoleSuspendBlockedIcon_.setVisible(false);
            consoleSuspendedIcon_.setVisible(false);
            break;
      }
   }

   @Override
   public void onSuspendedBlockedEvent(SessionSuspendBlockedEvent event) {
      consoleSuspendBlockedIcon_.setVisible(event.isBlocked());
      consoleSuspendBlockedIcon_.setTitle(event.getMsg());
   }
```